bridge/client/app/_views/ktb-integration-view/ktb-integration-view.component.html

```diff
@@ -59,18 +59,5 @@ <h2>Keptn API</h2>
         </dt-expandable-panel>
       </li>
     </ul>
-    <p *ngIf="!integrationsExternalDetails">
-      <button dt-button (click)="loadIntegrations()">
-        <dt-icon name="xhr-load-action"></dt-icon> Load more information about integrations
-      </button>
-    </p>
-    <ktb-markdown [html]="integrationsExternalDetails"></ktb-markdown>
```

Comment:
Since `ktb-markdown` is no longer used, please also remove the component and the dependency to `highlight.js`

Reply:
Should I delete the ktb-markdown folder?